content/en/synthetics/ci.md

```diff
@@ -17,15 +17,12 @@ further_reading:
   text: "Configure an API Test"
 ---
 
-<div class="alert alert-warning">
-This feature is in private beta. To request access, contact <a href="/help/">Datadog Support</a>.
-</div>
-
-On top of executing your tests at predefined intervals, you can also execute Datadog Synthetic tests on-demand using the dedicated API endpoints. You can execute Datadog Synthetic tests in your continuous integration (CI) pipelines, enabling you to block the deployment of branches which would break your key features and endpoints.
+On top of executing your tests at predefined intervals, you can also **execute Datadog Synthetic tests on-demand** using the dedicated API endpoints. You can execute **Datadog Synthetic tests in your continuous integration (CI) pipelines**, enabling you to block the deployment of branches which would break your key features and endpoints. 
```

Comment:
```suggestion
In addition to running tests at predefined intervals, you can also run Datadog Synthetic tests on-demand using API endpoints. You can run Datadog Synthetic tests in your continuous integration (CI) pipelines, letting you block the deployment of branches that would break your product. 
```